package.json

```diff
@@ -7,24 +7,25 @@
     "node": ">=10.11.0 <12"
   },
   "resolutions": {
-    "**/@types/node": "~10.3.6",
+    "**/@types/node": "~10.12.0",
```

Comment:
try to keep old node version if getting Node.js 12 is not possible next week

Reply:
cc @marcdumais-work there are different deps update, but i think we are better postpone licenses check before we settle on the final version of the PR

Reply:
I reverted it to current Node version